src/platforms/apple/common/install/swift-package-manager.mdx

```diff
@@ -3,12 +3,6 @@ title: Swift Package Manager (SPM)
 description: "Integrate Sentry into your Xcode project using Swift Package Manager (SPM)."
 ---
 
-To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Swift Packages` -> `Add Package Dependency`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or branch) on the next page.
+To integrate Sentry into your Xcode project using Swift Package Manager (SPM), open your App in Xcode and open `File` -> `Add Packages`. Then add the SDK by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` on the top-right search field. Define you dependecy rule by selecting the SDK version (or branch), then press "Add Package" button.
 
-When using SPM version 5.2 or later, you must specify the the target name too:
-
-```swift {tabTitle:Swift}
-.package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa", from: "{{ packages.version('sentry.cocoa') }}"),
-```
-
-Version tags or branches need to have the `Package.swift` file in it or Xcode won't be able to install the package.
+NOTE: Version tags or branches need to have the Package.swift file in it or Xcode won't be able to install the package. Thus versions previous to `4.4.3` can't be installed via SPM!
```

Comment:
We avoid exclamation points in the docs, so I've spelled out the word "cannot" instead of having it be a contraction and bolded it to make it more prominent. Also I've styled this as a note. 

```suggestion
<Note>

Version tags or branches need to have the Package.swift file in it or Xcode won't be able to install the package. This means versions previous to `4.4.3` *cannot* be installed using SPM.
 
 </Note>
```